dev/NavigationView/NavigationViewItemAutomationPeer.cpp

```diff
@@ -199,7 +206,7 @@ int32_t NavigationViewItemAutomationPeer::GetPositionOrSetCountInLeftNavHelper(A
     
     if (auto navview = GetParentNavigationView())
     {
-        if (auto listview = winrt::get_self<NavigationView>(navview)->LeftNavListView())
+        if (auto repeater = winrt::get_self<NavigationView>(navview)->LeftNavRepeater())
```

Comment:
>repeater [](start = 17, length = 8)

locals should be const where possible as per c++ guidelines con.4 [https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction)